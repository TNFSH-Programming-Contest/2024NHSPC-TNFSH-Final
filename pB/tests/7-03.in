7 12
3 8 5 2 1 9 6
3602459 1 2 11338 477335 26421 252 896137427 23761467 28186 1180468 499436
0 
7 6 2 9 12 10 8 1
100
3 6 9 5 8 2 10
1 3
68
3 3 2 4 5 7 10
9 12 1 5 8 11 6 7 9 4
61
4 1 1 5 2 3 7 4 4
5 11 10 3 1 8
56
7 4 7 5 10 1 3 3 2 7 5 6 1 2 5
3 6 10 8
31
0 
7 11 8 7 9 10 1 12
18
5 1 5 6 1 3 7 4 3 7 4
7 2 8 5 1 3 7 12
2
6 6 8 1 3 7 1 4 6 2 5 3 6
6 1 9 12 3 8 10
94
1 3 8
0 
66
6 6 3 1 8 7 7 5 9 2 5 4 7
9 6 1 9 12 10 5 2 11 7
89
0 
5 3 9 12 7 11
11
5 1 1 2 3 5 4 7 4 3 4
4 9 8 5 2
46
0 
8 8 4 2 5 6 3 7 1
37
7 3 6 2 8 7 8 1 10 4 10 6 8 5 9
5 8 12 5 9 7
55
2 7 5 1 6
7 1 12 8 4 7 3 5
10
5 5 9 3 2 7 8 1 1 2 3
9 2 5 9 4 7 12 1 8 3
64
5 3 2 5 3 7 6 6 1 1 2
8 2 5 8 11 7 10 12 1
56
2 2 1 3 6
6 10 2 4 12 3 7
86
0 
7 4 2 6 1 12 10 5
12
1 2 10
0 
0
0 
8 8 11 3 10 9 12 4 7
36
3 4 6 7 5 5 8
7 10 8 6 5 9 4 7
28
3 5 8 2 9 6 3
1 10
35
4 3 4 7 2 4 4 5 3
9 3 4 5 8 10 6 1 2 12
72
2 1 6 4 3
2 3 6
74
2 2 8 5 7
1 6
59
2 1 3 5 3
3 8 12 6
24
1 6 2
6 4 11 3 6 10 1
98
0 
9 8 4 3 5 10 9 11 12 1
48
3 4 6 6 7 7 8
8 3 9 10 4 8 7 5 12
27
0 
5 5 9 4 6 2
13
2 3 1 1 8
9 12 10 5 3 7 1 4 8 6
95
1 2 8
4 1 8 3 11
50
2 6 2 3 3
8 3 10 7 6 8 12 2 1
25
