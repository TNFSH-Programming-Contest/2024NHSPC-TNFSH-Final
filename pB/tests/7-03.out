29588629
25441636
900255272
900783527
4645188
25441735
24276575
